manifest/me.art0007i/LargerObjectMessagePreviews/info.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "LargerObjectMessagePreviews",
+    "id": "me.art0007i.LargerObjectMessagePreviews",
+    "description": "Increases the size of object message thumbnails.",
+    "category": "Dashboard",
+    "sourceLocation": "https://github.com/art0007i/LargerObjectMessagePreviews",
+    "versions": {
+        "2.0.0": {
```

Comment:
Does not match the Version string in the compiled mod along the AssemblyVersion.
Looks like you did make these changes in your repo, they just were after your last compile

Reply:
you're totally right. going to fix it in a sec